File: config/config-sbc2ha/src/main/java/com/dfi/sbc2ha/config/sbc2ha/definition/actuator/McpOutputConfig.java
package com.dfi.sbc2ha.config.sbc2ha.definition.actuator;

import com.dfi.sbc2ha.config.sbc2ha.definition.enums.OutputKindType;
import lombok.Data;
import lombok.EqualsAndHashCode;

@EqualsAndHashCode(callSuper = true)
@Data
public class McpOutputConfig extends ActuatorConfig {

    public McpOutputConfig() {
        super();
        kind = OutputKindType.MCP;
    }
}

File: config/editor/src/app/definition/actuator/McpOutputConfig.ts
import {ActuatorConfig} from "./ActuatorConfig";
import {OutputKindType} from "../enums/OutputKindType";
import {ActuatorType} from "../enums/ActuatorType";


export class McpOutputConfig extends ActuatorConfig {
    constructor(name: string, outputType: ActuatorType, output: number) {
        super(OutputKindType.MCP, name, outputType, output);
    }
}
